src/main/java/com/epam/ta/reportportal/core/configs/EmailConfiguration.java

```diff
@@ -37,6 +40,11 @@
 	//		return new MailServiceFactory(templateEngine, encryptor, settingsRepository);
 	//	}
 
+	@Bean
+	public ExecutorService emailExecutorService() {
+		return Executors.newSingleThreadExecutor();
```

Comment:
can be changed to Spring's TaskExecutor (thread-pooling) so in future we can use it for different notifications